table.go

```diff
@@ -406,6 +407,63 @@ func (t *Table) SchemaIterator(
 	return err
 }
 
+func (t *Table) ArrowSchema(
```

Comment:
I don't think this interface is going to work out. I think we should change the interface to read to something like:

```
func (t *Table) View(func(txn *Txn) error) error
```

Then based on that transaction the schema could be read within the read-only transaction. Then a consistent view of schema and data could be guaranteed.